platform/ios/MGLMapView.mm

```diff
@@ -563,16 +562,7 @@ - (void)updateConstraints
 // This is the delegate of the GLKView object's display call.
 - (void)glkView:(GLKView *)view drawInRect:(CGRect)rect
 {
-    if (self.glSnapshotView)
-    {
-        NSUInteger snapshotIdx = [self.subviews indexOfObject:self.glSnapshotView];
-        NSUInteger glIdx = [self.subviews indexOfObject:self.glView];
-        NSAssert(snapshotIdx > glIdx,
-                 @"The snapshot view is the %luth subview, behind the GL view, which is the %luth. "
-                 @"The snapshot view should obscure the GL view to prevent crashing while the app is in the background.",
-                 (unsigned long)snapshotIdx, (unsigned long)glIdx);
-    }
-    else
+    if ( ! self.glSnapshotView || self.glSnapshotView.isHidden)
```

Comment:
@incanus When you use dot syntax, you shouldn’t use the getter explicitly. (Try code completion in Xcode, or try using `.isHidden` in Swift.)